lib/twingly/url_cache.rb

```diff
@@ -5,26 +5,40 @@
 
 module Twingly
   class UrlCache
+    class Error < StandardError; end
+    class ServerError < Error; end
+
     attr_reader :ttl
 
     CACHE_VALUE = ""
 
     def initialize(ttl: 0)
```

Comment:
I think we could have better specs if we do this:

```ruby
 def initialize(ttl: 0, cache: default_cache)
  @cache = cache
  ...
end

def default_cache
  Dalli::Client.new(servers, options)
end
```

Hopefully we can kill of many `any_instance_of` as we can inject our own test double where applicable.

Reply:
Doing it that way would get rid of all `any_instance_of` I think. I'll give it a try.

Reply:
cc06220